cinch/roles/certificate_authority/tasks/main.yml

```diff
@@ -1,7 +1,20 @@
-- name: install certificate authority
+- name: install certificate authority from URL
   get_url:
     url: "{{ item }}"
     dest: /etc/pki/ca-trust/source/anchors/
+    owner: root
+    group: root
+    mode: 0644
     validate_certs: "{{ certificate_authority_validate_certs }}"
   with_items: "{{ certificate_authority_urls }}"
   notify: install certificate authority
```

Comment:
Is it OK to have site.yml take care of this for us?

https://github.com/RedHatQE/cinch/blob/master/cinch/site.yml#L15

IIRC, most things in cinch require root and we just usually add a **become: false** for the few places that don't need it.